README_ADMIN.md

```diff
@@ -0,0 +1,218 @@
+rebuilt during a server migration. See [this section](#building-content) for details
+on when a content build might be necessary.
+
+By default, the `rsconnect-admin content search` command will return metadata for ALL
+of the content on a RStudio Connect server, both published and unpublished content.
+
+```bash
+# only published content
+$ rsconnect-admin content search --published
+
+# only unpublished content
+$ rsconnect-admin content search --unpublished
+
+# published content where the python version is at least 3.9.0
+$ rsconnect-admin content search --published --py-version ">=3.9.0"
+
+# published content where the R version is exactly 3.6.3
+$ rsconnect-admin content search --published --r-version "==3.6.3"
+
+# published content where the content type is a static RMD
+$ rsconnect-admin content search --content-type rmd-static
+
+# published content where the content type is either shiny OR fast-api
+$ rsconnect-admin content search --content-type shiny --content-type python-fastapi
+
+# any content, published or unpublished, where the title contains the text "Stock Report"
+$ rsconnect-admin content search --title-contains "Stock Report"
+
+# published content, results are ordered by when the content was last deployed
+$ rsconnect-admin content search --published --order-by last_deployed
+
+# published content, results are ordered by when the content was created
+$ rsconnect-admin content search --published --order-by created
+```
+
+
+### Building Content
+
+RStudio Connect caches R and Python packages in the configured
+[`Server.DataDir`](https://docs.rstudio.com/connect/admin/appendix/configuration/#Server.DataDir).
+Sometimes, these package caches become stale and need to be rebuilt. This refresh
+occurs automatically when an RStudio Connect user visits the content via the
+RStudio Connect dashboard. Some content, however, is high priority or not accessed
+frequently via the dashboard (API content, emailed reports). In these cases, administrators
+can preemptively build specific content items using the `rsconnect-admin build` subcommands.
+This way the user does not have to pay the build cost when the content is accessed next.
+
+The following are some common scenarios where performing a content build might be necessary:
+
+- OS upgrade
+- changes to gcc or libc libraries
+- changes to Python or R installations
+- switching from source to binary package repositories or vice versa
+- migrating from local content execution to remote content execution with Kubernetes
+
+Administrators should use the [`rsconnect-admin content search`](#search) subcommand to help
+identify content items for building.
+
+To build a specific content item, first `add` it to the list of content that is
+"tracked" for building using its GUID.
+
+```bash
+# add a single content item by guid
+$ rsconnect-admin build add --guid 4ffc819c-065c-420c-88eb-332db1133317
+```
+
+Administrators can view currently "tracked" content items using the
```

Comment:
It feels like this is a sequence.. but maybe it isn't? If you intended it to be, then you might consider a numeric list for the steps. If it isn't, then perhaps find a way to describe each subcommand in isolation? (I think I'm being thrown off by the `first add it...` words above.

If it is a sequence, I'd re-order, putting run right after add, which would move ls downward to use after building has been initiated. Unless you want to ls twice.. once before starting and then again after, with different options.